### DISCUSSION TITLE
Are password protected pages possible?

### DISCUSSION THREAD

**Discussion by pftnhr** [YOURLS/YOURLS#3310] [Ideas]
I was just wondering if there is a way to access pages only after a successful login. 

The background is that I only want to allow users with login to see the form for creating the Short URLs, but do not want them in the backend.

**Comment by ozh** [comment]
"Pages" when talking about YOURLS refer to what lives in the `user/pages/` folder, ie http://yourls.org/examplepage / [user/pages/examplepage.php](https://github.com/YOURLS/YOURLS/blob/master/user/pages/examplepage.php)

What you're describing would be like the example "public interface" that ships with YOURLS, but password protected.

Easy peasy:
- copy `sample-public-front-page.txt` to `sample-public-front-page.php`
- add `yourls_maybe_require_auth();` right after the `require_once( dirname(__FILE__).'/includes/load-yourls.php' );` in `sample-public-front-page.php`

**Reply by pftnhr** [reply]
Ha! That's as easy peasy as I wanted it. 😃

Thanks a lot!
